python/cudf/cudf/tests/test_groupby.py

```diff
@@ -908,7 +908,7 @@ def test_groupby_dropna():
         {"a": [1, 1, 1, None], "b": [1, None, 1, None], "c": [1, 2, 3, 4]}
     )
     idx = cudf.MultiIndex.from_frame(
-        df[["a", "b"]].drop_duplicates(), names=["a", "b"]
+        df[["a", "b"]].drop_duplicates().sort_values(["a", "b"]), names=["a", "b"]
     )
     expect = cudf.DataFrame({"c": [4, 2, 4]}, index=idx)
```

Comment:
may want to use a sort_index here instead of sorting the multiindex above for consistency